plugins/ngrok/provisioner.go

```diff
@@ -28,12 +28,12 @@ const (
 type fileProvisioner struct {
 }
 
-func ngrokProvisioner() sdk.Provisioner {
+func IsNgrokAPIKeySupported() bool {
```

Comment:
Let's call this function something clearer, such as `HasEnvVarSupport` or something like that.

Reply:
Done!